CMakeLists.txt

```diff
@@ -74,15 +74,18 @@ option(use_prov_client "Enable provisioning client" OFF)
 option(use_tpm_simulator "tpm simulator type of hsm used with the provisioning client" OFF)
 option(use_edge_modules "Enable support for running modules against Azure IoT Edge" OFF)
 option(use_custom_heap "use externally defined heap functions instead of the malloc family" OFF)
-option(use_uhttp_upload "set use_uhttp_upload to ON to use uhttp for upload to blob" OFF)
+option(use_uhttp_upload "set use_uhttp_upload to ON to use uhttp for upload to blob" ON)
```

Comment:
Is this really going to be the default going forward?

Reply:
Yes, for this branch.